src/deps.ts

```diff
@@ -394,14 +394,21 @@ export interface AutoEncryptionOptions {
   };
 }
 
-/** @public */
-export interface AutoEncrypter {
-  // eslint-disable-next-line @typescript-eslint/no-misused-new
-  new (client: MongoClient, options: AutoEncryptionOptions): AutoEncrypter;
-  init(cb: Callback): void;
-  teardown(force: boolean, callback: Callback): void;
-  encrypt(ns: string, cmd: Document, options: any, callback: Callback<Document>): void;
-  decrypt(cmd: Document, options: any, callback: Callback<Document>): void;
-  /** @experimental */
-  readonly cryptSharedLibVersionInfo: { version: bigint; versionStr: string } | null;
+type MongoCrypt = { MongoCrypt: any };
+/** A utility function to get the instance of mongodb-client-encryption, if it exists. */
+export function getMongoDBClientEncryption(): MongoCrypt | null {
+  let mongodbClientEncryption = null;
+
+  try {
+    // NOTE(NODE-3199): Ensure you always wrap an optional require literally in the try block
+    // Cannot be moved to helper utility function, bundlers search and replace the actual require call
+    // in a way that makes this line throw at bundle time, not runtime, catching here will make bundling succeed
+    mongodbClientEncryption = require('mongodb-client-encryption');
```

Comment:
What is the reason for `MONGODB_CLIENT_ENCRYPTION_OVERRIDE` env variable support being removed here?

Reply:
from slack: This was for testing b/c there was a circular dependency that cause a BF https://jira.mongodb.org/browse/NODE-4254, there's no longer a driver dep in FLE.